src/sandstorm/run-bundle.c++

```diff
@@ -1822,6 +1825,7 @@ private:
           ", \"allowDevAccounts\":", config.allowDevAccounts ? "true" : "false",
           ", \"isTesting\":", config.isTesting ? "true" : "false",
           ", \"wildcardHost\":\"", config.wildcardHost, "\"",
+          ", \"sandcatsHostname\":\"", config.sandcatsHostname, "\"",
```

Comment:
I guess you're relying on Javascript treating empty strings as falsy, but it seems safer to not include this field or explicitly set it null if sandcatsHostname isn't set...